src/software/ai/hl/stp/play/penalty_kick_play.cpp

```diff
@@ -68,19 +53,25 @@ void PenaltyKickPlay::getNextTactics(TacticCoroutine::push_type &yield)
 
         // Move all non-shooter robots to the center of the field
         move_tactic_2->updateControlParams(Point(0, 0),
-                                           world.field().enemyGoal().orientation(), 0);
+                                           world.field().enemyGoal().orientation(), 0,
+                                           BallNavigationType::AVOID_COLLISION);
```

Comment:
I thought ball navigation was only going to be handled by tactics and not exposed to plays? Or am I mis-remembering?

Reply:
Plays can change BallCollisionType but not motion constraints. BallCollisionType is an exception to the gamestate + tactic -> motionconstraint mapping

Reply:
From #793 I thought we decided
>MoveIntent will have a flag for touching / not touching the ball, because there is semantic information in the high level about whether to touch the ball or not

Which made me think only `Actions` would have control over ball avoidance, and not Plays. Or is that different from `BallCollisionType`?

Reply:
I suppose it's ambiguous in that quote what high level means. Does it extend up to actions or all the way up to plays? Or does it go up to tactics?

Reply:
I didn't think it would go up to plays, since we want to remove that complexity from plays and prevent them from dictating so much behaviour that is robot / task specific, which is what tactics and actions do. I'm fine if you don't change this in this PR, but can you open a ticket for discussion on this again?

Reply:
Was there a ticket opened for this?